CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ##### Enhancements
 
+##### Bug Fixes
+
+* None.
+
+
+## 2.0.1 (2019-04-30)
```

Comment:
We're releasing version `2.1.0`, right? (not `2.0.1`)

Reply:
I'll fix that typo.